src/components/App.tsx

```diff
@@ -1016,7 +1026,10 @@ function App() {
               bottomLeft: false,
               topLeft: false,
             }}
-            onResizeStop={autoZoomIfNecessary}
+            onResizeStop={(_, __, ref) => {
+              setSavedChatSize(ref.style.width);
+              autoZoomIfNecessary();
+            }}
```

Comment:
now that i think of it lets add a mixpanel event here

Reply:
added!